modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-prepare-cloud-credentials-gcp-non-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} on GCP
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP cluster you must create a `CredentialsRequest` object and allow the Cloud Credential Operator to generate the cloud credentials secret.
+
+.Prerequisites
+
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  generation: 1
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
```

Comment:
Not needed.

Reply:
@xenolinux If you end up removing the metadata.labels in the CredentialsRequest, please do the same in the other 3 PRs as well.